TombEngine/Objects/Effects/enemy_missile.cpp

```diff
@@ -305,7 +305,7 @@ namespace TEN::Entities::Effects
 				}
 			}
 			else
-				TriggerShockwave(&fx->pos, 24, 88, 48, 64, 128, 0, 16, (((~g_Level.Rooms[fx->roomNumber].flags) / 16) & 2) * 65536, 0);
+				TriggerShockwave(&fx->pos, 24, 88, 48, 64, 128, 0, 16, (((~g_Level.Rooms[fx->roomNumber].flags) / 16) & 2) * 65536, 2);
```

Comment:
What `2` exactly means? This argument supposed to be some kind of flags. If possible, replace with enum.

Reply:
I made an enum for damage argument